src/extension/providers/liquid/index.ts

```diff
@@ -17,7 +17,11 @@ export default class LiquidProvider {
     if (this.enabled) {
       return { enabled: true };
     }
-    return await this.execute("enable");
+    const result = await this.execute("enable");
+    if (typeof result.enabled === "boolean") {
+      this.enabled = result.enabled;
```

Comment:
So I have to call `enable` to init the value? it means I can't show the enable state to the user without any interaction (calling `enable` may show a popup if it's the first time). 

is it possible to recover the value of "enabled" directly in the constructor? or have an async getter  `isEnabled()` just checking if the user has already called `enable` function in the past ?

Reply:
@louisinger that's a good point. I will see if there is a way that can be done.

Reply:
@louisinger I chatted with @reneaaron and we think enable() should always be called after a user interaction (e.g. button click), it could be up to an application itself to store whether or not the user has ever logged in, to call enable immediately on page load.